### PR TITLE
Update Spanish translations

### PR DESCRIPTION
**Pull request by OscarSantosMu**
#### Summary

Most of the changes were made at the [README](https://github.com/education/codespaces-teaching-template-py/blob/main/translations/es/README.md), particularly I added the whole `🤖 Use Copilot to create a lesson for students`  section, update the previous sections and fix minor syntax issues at the [Jupyter Notebook](https://github.com/education/codespaces-teaching-template-py/blob/main/translations/es/example-notebook.ipynb) and the [slides](https://github.com/education/codespaces-teaching-template-py/blob/main/translations/es/slides_es.pptx)

***

#### 🎓 **Student Ambassador**

***

### CODE REVIEW

File: translations/es/README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 Con esta plantilla puedes crear rápidamente un entorno normalizado para enseñar o aprender Python. Haz que tus estudiantes se centren en su aprendizaje en lugar de configurar su entorno. Esta plantilla utiliza Codespaces, un entorno de desarrollo alojado en la nube con [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), un poderoso editor de texto.
 
+Tú también tendrás la oportunidad de probar Copilot para crear un plan de clase utilizando el archivo [example-lesson.ipynb](./example-lesson.ipynb).
+
 🤔 ¿Curioso? Mira el siguiente vídeo donde te explicamos todos los detalles:
 
-[![Enseñando Python con Codespaces](../../images/video-banner.gif)](https://youtu.be/7rMvb03hHpI "Teaching Python with Codespaces")
+[![Enseñando Python con Codespaces](https://img.youtube.com/vi/7rMvb03hHpI/0.jpg)](https://youtu.be/7rMvb03hHpI "Teaching Python with Codespacess")
```
Comment (cyz) [inline translations/es/README.md:17]: you can translate to "Enseñando Python con Codespaces (En inglés)

File: translations/es/README.md [hunk 2]
```diff
@@ -51,28 +53,25 @@ Puedes configurar:
 
 Este repositorio es una plantilla de GitHub, la cual contiene lo siguiente:
 
-- [example-notebook.ipynb](./example-notebook.ipynb), Este notebook utiliza la librería [Pandas](https://pandas.pydata.org/) para enseñar operaciones básicas con un pequeño archivo CSV (_Comma Separated Value_) [dataset](./wine-regions.csv)
-- [.devcontainer/Dockerfile](./.devcontainer/Dockerfile), para que pueda configurar qué sistema operativo utilizará el Codespace y cómo se debe construir el contenedor.
-- [.devcontainer/devcontainer.json](./.devcontainer/devcontainer.json), Un archivo de configuración utilizado por Codespaces para configurar [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), por ejemplo, para agregar y habilitar una extensión.
+- [example-notebook.ipynb](./example-notebook.ipynb): Un notebook que utiliza la librería [Pandas](https://pandas.pydata.org/) para enseñar operaciones básicas con un pequeño archivo CSV (_Comma Separated Value_) [dataset](./wine-regions.csv)
+- [.devcontainer/Dockerfile](./.devcontainer/Dockerfile): Archivo de configuración usado por Codespaces para determinar el sistema operativo y otros detalles.
```
Comment (cyz) [inline translations/es/README.md:57]: here you can update to navigate to the .devcontainer folder in the main folder or move the .devcontainer folder to the es

File: translations/es/README.md [hunk 3]
```diff
@@ -51,28 +53,25 @@ Puedes configurar:
...
+- [.devcontainer/devcontainer.json](./.devcontainer/devcontainer.json): Un archivo de configuración utilizado por Codespaces para configurar [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), por ejemplo, para agregar y habilitar extensiones adicionales.
```
Comment (cyz) [inline translations/es/README.md:58]: same comment above

File: translations/es/README.md [hunk 4]
```diff
@@ -51,28 +53,25 @@ Puedes configurar:
...
 ## 🧐 ¡Pruébalo!
 
 Prueba este repositorio de plantillas con Codespaces siguiendo estos pasos:
 
-1. Crea un repositorio desde esta plantilla. Utiliza este link [crea un repositorio](https://github.com/microsoft/codespaces-teaching-template-py/generate)
-1. Ve a la página principal del repositorio recién creado.
-1. Debajo del nombre del repositorio, usa el menú desplegable Código y, en la pestaña Codespaces, selecciona "Crear Codespace en main".
+1. Crea un repositorio desde esta plantilla. Utiliza este [link crea un repositorio](https://github.com/microsoft/codespaces-teaching-template-py/generate). Tú puedes hacer este repositorio privado o público, es tú decisión.
+2. Navega a la página principal del repositorio recién creado.
+3. Debajo del nombre del repositorio, usa el menú desplegable Código, y en la pestaña Codespaces, selecciona "Crear Codespace en main".
```
Comment (cyz) [inline translations/es/README.md:67]: Suggestion: Debajo del nombre del repositorio, usa el menú desplegable Code (Código), y en la pestaña Codespaces, selecciona "Create codespace on main" (Crear Codespace en main).

File: translations/es/README.md [hunk 5]
```diff
@@ -51,28 +53,25 @@ Puedes configurar:
...
    ![Crea un codespace](https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png)
-1. Se empieza a crear tu Codespace
+4. Espera mientras se inicializa tu Codespace
```
Comment (cyz) [inline translations/es/README.md:69]: This step change a bit in english. You can consider "Espera mientras GitHub inicializa el Codespace."

File: translations/es/README.md [hunk 6]
```diff
@@ -145,10 +144,98 @@ Para encontrar el identificador único de una extensión:
 - Ingresa a la página web de la extensión, por ejemplo [https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter&WT.mc_id=academic-77460-alfredodeza)
 - Localiza el campo *Unique Identifier* bajo la sección **More info** en tu lado derecho.
 
+## 🤖 Utiliza Copilot para crear una lección para alumnos
+GitHub Copilot ya está disponible en GitHub Codespaces. Puedes utilizar Copilot para crear una clase para tus alumnos. Este repositorio incluye la extensión de Copilot para que puedas utilizarla inmediatamente. Asegúrate de que tu cuenta tiene acceso a Copilot. Si no tienes acceso, puedes [solicitar acceso aquí](https://github.com/login?return_to=%2Fgithub-copilot%2Fsignup).
+
+GitHub Copilot es gratis para estudiantes y profesores. [Más información](https://education.github.com/pack/offers). Sigue [estos pasos](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279?WT.mc_id=academic-0000-alfredodeza) para verificar tu afiliación como estudiante o profesor y activar Copilot de forma gratuita.
```
Comment (cyz) [inline translations/es/README.md:150]: please, include the spanish article in the tech community link: https://aka.ms/GitHubCopilot4Edu

File: translations/es/example-notebook.ipynb [hunk 7]
```diff
@@ -1,12 +1,13 @@
...
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "# Operaciones básicas de Pandas\n",
     "\n",
-    "Este cuaderno te guiará a través de algunas operaciones básicas con la biblioteca De Python de Pandas para explorar un pequeño conjunto de datos de regiones vinícolas de todo el mundo.\n",
+    "Este cuaderno te guiará a través de algunas operaciones básicas con la librería Pandas de Python para explorar un pequeño conjunto de datos de regiones vinícolas de todo el mundo.\n",
     "\n",
     "**Objetivos de aprendizaje:**\n",
     "\n",
```
Comment (cyz) [inline translations/es/example-notebook.ipynb:13]: we need fix the list to 1, 2, 3, etc

File: translations/es/example-notebook.ipynb [hunk 8]
```diff
@@ -451,6 +455,7 @@
    ]
```
Comment (cyz) [inline translations/es/example-notebook.ipynb:455]: need include the section "Use plotting for visual context" https://github.com/OscarSantosMu/codespaces-teaching-template-py/blob/main/example-notebook.ipynb

**Review by cyz — CHANGES_REQUESTED** [review]
Hola @OscarSantosMu gracias por tu trabajo. He enviado algunos comentarios de ajustes. Avísame si tienes alguna duda.

**Comment by cyz** [comment]
Hola @OscarSantosMu necesitas ayuda?

File: translations/es/example-notebook.ipynb (hunk 7 above)
Comment (OscarSantosMu) [reply translations/es/example-notebook.ipynb:13]: What do you mean?

File: translations/es/example-notebook.ipynb (hunk 8 above)
Comment (OscarSantosMu) [reply translations/es/example-notebook.ipynb:455]: Can you provide more details? The only update I did in the [translations/es/example-notebook.ipynb](https://github.com/education/codespaces-teaching-template-py/pull/41/files/2225b078826cee9a64e577de2fd9a810756b92eb#diff-fd259aaa37fc2a64cf08eec1233ffcd3efecf51acfa0901db8f77171cfb23dbe) was a single rephrase at the "Operaciones básicas de Pandas" paragraph.

**Comment by OscarSantosMu** [comment]
Sorry @cyz, I was off for a couple of days. I made most of the changes. Can you just clarify those from the notebook please?

File: translations/es/example-notebook.ipynb (hunk 8 above)
Comment (cyz) [reply translations/es/example-notebook.ipynb:455]: Hi @OscarSantosMu include a new section named "Use plotting for visual context". You can check the new section and compare with your file (I'm linking the notebook in english): https://github.com/OscarSantosMu/codespaces-teaching-template-py/blob/main/example-notebook.ipynb

Comment (OscarSantosMu) [reply translations/es/example-notebook.ipynb:455]: I do not get it, the section is already there for me.

image

left (spanish section) - right (linked english section)

And the section is also mentioned at the Learning Objectives (Objetivos de aprendizaje) as well.

Comment (alfredodeza) [reply translations/es/example-notebook.ipynb:455]: This looks OK to me @cyz . Thanks @OscarSantosMu

**Review by alfredodeza — APPROVED** [review]

File: translations/es/example-notebook.ipynb (hunk 8 above)
Comment (cyz) [reply translations/es/example-notebook.ipynb:455]: thank you @alfredodeza for you review and @OscarSantosMu :) all good!

**Pull request merged**